src/scripts/main.js

```diff
@@ -1,3 +1,7 @@
 'use strict';
 
-// write code here
+const wall = document.querySelector('.wall');
+const spider = document.querySelector('.spider');
+
+spider.style.top = `${((wall.offsetHeight - spider.offsetHeight) / 2)}px`;
+spider.style.left = `${((wall.offsetWidth - spider.offsetWidth) / 2)}px`;
```

Comment:
тут потрібно виправити, щоб павук був по центру стіни

```suggestion
spider.style.cssText = `
  top: ${wall.clientHeight / 2 - spider.clientHeight / 2}px;
  left: ${wall.clientWidth / 2 - spider.clientWidth / 2}px`;
```
щось типу того